### PR TITLE
zh-translation: /zh/docs/setup/upgrade/gateways/index.md

### PR DESCRIPTION
**Pull request by MXuDong**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/setup/upgrade/gateways/index.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
+---
+title: 多个修正版的网关管理 [实验性]
```
Comment (kebe7jun) [inline content/zh/docs/setup/upgrade/gateways/index.md:2]:
```suggestion
title: 多版本的 Gateway 管理 [实验性]
```

File: content/zh/docs/setup/upgrade/gateways/index.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+description: 使用网关配置和升级 Istio (实验性)。
```
Comment (kebe7jun) [inline content/zh/docs/setup/upgrade/gateways/index.md:3]: Gateway

File: content/zh/docs/setup/upgrade/gateways/index.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+weight: 30
+keywords: [kubernetes,upgrading,gateway]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+{{< boilerplate experimental >}}
+
+使用一个 `IstioOperator` CR，即使在使用 [Canary 控制平面的方式](/zh/docs/setup/upgrade/canary)，CR 中定义的任何网关（包括安装在默认配置文件中的 `istio-ingressgateway`）也会被热升级。但应该避免这样，因为网关是影响应用程序正常运行时的关键组件。在新的控制和数据平面可以正常工作以后，再升级网关。
```
Comment (kebe7jun) [inline content/zh/docs/setup/upgrade/gateways/index.md:12]: 这个可以叫金丝雀升级。
https://github.com/istio/istio.io/pull/9196

File: content/zh/docs/setup/upgrade/gateways/index.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+1.  安装 `CR`：
+
+    {{< text bash >}}
+    $ istio-1.8.0/bin/istioctl install -n istio-system -f control-plane.yaml --revision 1-8-0
+    $ istio-1.8.0/bin/istioctl install -n istio-system -f gateways.yaml --revision 1-8-0
+    {{< /text >}}
+
+Istioctl 的安装和操作动作通过 revision 和拥有者的 CR 名称进行确定具有所有权的资源。只有传递给 `istioctl` 安装、操作资源的名称和 revision 标签可以被 `IstioOperator` CR 匹配时，该资源才会受到 CR 更改的影响，集群内的其他资源都将被忽略。注意确保每个 `IstioOperator` 安装的组件不会与另一个 `IstioOperator` CR 相互重叠，否则两个 CR 会导致控制器或 `istioctl` 命令相互干扰。
+
+### 使用 `Istioctl` 升级 {#upgrade-with-istioctl}
```
Comment (kebe7jun) [inline content/zh/docs/setup/upgrade/gateways/index.md:63]: istioctl 不用大写。

**Review by kebe7jun — COMMENTED** [review]
再核对核对。

File: content/zh/docs/setup/upgrade/gateways/index.md (hunk 3 above)
Comment (MXuDong) [reply content/zh/docs/setup/upgrade/gateways/index.md:12]: OK

File: content/zh/docs/setup/upgrade/gateways/index.md (hunk 4 above)
Comment (MXuDong) [reply content/zh/docs/setup/upgrade/gateways/index.md:63]: OK

File: content/zh/docs/setup/upgrade/gateways/index.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+    istiod-1-8-0-74f95c59c-4p6mc    1/1     Running   0          68m
+    istiod-1-8-1-65b64fc749-5zq8w   1/1     Running   0          13m
+    {{< /text >}}
+
+1.  有关工作负载过渡到新的 Istio 版本的更多详细信息，请查阅金丝雀升级文档。
+
+    -  使用 istio.io/rev=1-8-1 标记工作负载的命名空间，并且重启相应的工作负载。
+    -  验证工作负载是否已经注入新的代理版本，并且集群已经正常运行。
+
+1.  将 Gateway  升级到新版本。在安装步骤中，编辑 `Gateway .yaml` 文件，将所有的 `1-8-0` 替换为 `1-8-1` 的版本，并重新部署该文件：
```
Comment (kebe7jun) [inline content/zh/docs/setup/upgrade/gateways/index.md:188]: Gateway .yaml?
是不是错了？

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**